tree.go

```diff
@@ -32,7 +40,8 @@ func NewDefaultTree() Tree {
 	}
 }
 
-func (d *DefaultTree) Push(data []byte) {
+func (d *DefaultTree) Push(data []byte, idx CellIndex) {
```

Comment:
It's common to prefix unused vars with `_` or you can simply use `_`.
```suggestion
func (d *DefaultTree) Push(data []byte, _idx CellIndex) {
```
or 
```go
func (d *DefaultTree) Push(data []byte, _ CellIndex) {
```